core/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala

```diff
@@ -70,7 +70,7 @@ class ToolTextFileWriter(
     // No need to close the outputStream.
     // Java should handle nested streams automatically.
     utf8Writer.foreach { writer =>
-      logInfo(s"$finalLocationText output location: $textOutputLoc")
+      logDebug(s"$finalLocationText output location: $textOutputLoc")
```

Comment:
I think it might be unnecessary but should we consider adding a single INFO level log per app indicating successful output generation, instead of logging for each individual file?

Again, it is unnecessary because these metrics are primarily for our internal use.

Reply:
Let me know if I get your comment.
Do you mean some sort of a summary of files that were generated but put them in a single message?
- If that's the case, then I think we will hit the same issue that we have on the python side and Qualx. For example, when we generated tree of the output; it became too long. So, I believe if we put a single LogInfo message that contains list of all metrics generated, then the list will keep growing and I don't see that very useful information to put in the logging message. 
- In that change I changed the debug-level so that it does not get generated by default in production.

Reply:
I was thinking of a single log line like:
```
INFO: Successfully generated output files for app <app-id> at <metrics location dir>
```

Reply:
Sure! For now this PR does exactly that.
We can later improve the implementation if we want the debug-level to be a single message as well.